server/app/views/admin/questions/QuestionPreviewFragment.html

```diff
@@ -1,5 +1,16 @@
 <div id="sample-question" class="margin-y-4">
+  <th:block th:if="${question.getType().getLabel() == 'File Upload'}">
+    <div
+      th:if="${question.getType().getLabel() == 'File Upload'}"
+      th:replace="~{questiontypes/FileUploadQuestionFragment :: file-upload(${question}, ${questionRendererParams})}"
+    ></div>
+  </th:block>
+
   <div
+    th:if="${question.getType().getLabel() != 'File Upload'}"
     th:replace="~{questiontypes/QuestionFragment :: applicant-question(${question}, ${questionRendererParams}, ${stateAbbreviations})}"
   ></div>
+  <div
+    th:replace="~{applicant/ApplicantBaseFragment :: pageFooterScripts}"
```

Comment:
**question (blocking):** Why do we need to add the footer?  Isn't this just a preview of the question itself?

Reply:
Oh, got it.  I don't think it needs a comment.  I just should have read more carefully and realized that this was the scripts.